x-pack/plugins/observability_solution/slo/public/application.tsx

```diff
@@ -116,15 +114,13 @@ export const renderApp = ({
                 }}
               >
                 <Router history={history}>
-                  <EuiThemeProvider darkMode={isDarkMode}>
```

Comment:
removing this deprecated theme provider, kept `<KibanaThemeProvider {...{ theme: { theme$ } }}>` from above